nCompiler/R/NC_Compile.R

```diff
@@ -82,7 +82,11 @@ nCompile_nClass <- function(NC,
   }
   interface <- match.arg(interface)
   if(interface == "generic")
-    return(newCobjFun)
+    return(function() {
+      ans <- newCobjFun()
+      ans$.NC <- NC
```

Comment:
TODO: Require an additional argument to `nDerivs()` / `setup_wrt()` for the original `nClass` generator rather than sticking the generator in here.

Reply:
See 152e9a6eecc330dfd48be52bc7323feb8309059b